docs/lib/examples/import-basic.js

```diff
@@ -1,19 +1,27 @@
-import React from 'react';
-import { Player } from 'cueplayer-react';
+import React from 'react'
+import {Player} from 'cueplayer-react'
+import {PlayerProvider} from 'cueplayer-react/context/player-context'
+import SidePanel from '../Components/side-panel/SidePanel'
 
 export default props => {
   return (
-    <Player
-      autoPlay
-      src="https://bitmovin-a.akamaihd.net/content/MI201109210084_1/MI201109210084_mpeg-4_hd_high_1080p25_10mbits.mp4"
-      crossOrigin="anonymous"
-    >
-      <track
-        id="notes"
-        src="https://gist.githubusercontent.com/joelhooks/bd3c1d68cb5a67adfcd6c035200d1fde/raw/aa7060f584e04db26c5fa6b464bf2058ed6f6e93/notes.vtt"
-        kind="metadata"
-        label="notes"
-      />
-    </Player>
-  );
-};
+    <PlayerProvider>
```

Comment:
for `Player` sibling components we need a provider to deliver context. `Player` accesses context and sets the player data so it can be accessed outside of the player.